src/routes/transactions/mappers/transactions-history.mapper.ts

```diff
@@ -115,8 +121,6 @@ export class TransactionsHistoryMapper {
       args.safe,
       args.onlyTrusted,
     );
-    // Remove first transaction that was requested to get previous day timestamp
-    args.transactionsDomain = args.transactionsDomain.slice(1);
```

Comment:
Question: AFAI understood, the original `transactionsDomain` array is not mutated here because the `slice` method creates a new array. That's the reason why there first transaction is not removed, Am I right?

Reply:
Correct.